frontend/src/services/ErrorService.ts

```diff
@@ -1,39 +0,0 @@
-import { useToast } from 'vue-toastification';
-import axios, { type AxiosError } from 'axios';
-
-function onError(error: any, info: string) {
-    console.error(`Error occurred: ${error.toString()}`);
-    const toast = useToast();
-    const genericErrorMsg =
-        'An application error has occurred. Please try again. If the error persists contact support.';
-
-    // Axios Http instance error that we like to pop out additional toast message.
-    if (axios.isAxiosError(error)) {
-        const err = error as AxiosError;
-        const axiosResponse = err.response;
-        const status = axiosResponse?.status;
-
-        const e401_authenticationErrorMsg =
-            'You are not logged in. Please log in.';
-        const e403_authorizationErrorMsg =
-            'You do not have the necessary authorization for the requested action.';
-        const e409_conflictErrorMsg = `${axiosResponse?.data.detail}`;
```

Comment:
Maybe previous **ErrorService** needs to be improved, but I wonder, if this ErrorService is now removed and in **main.ts** you also removed "**app.config.errorHandler**", how does the system now react to generic errors like above 401/403/409 etc? Does it mean, each component that connect to backend will need to handle them specifically or the system will have uncaught errors on console and won't show messages to the users?

Reply:
Sorry, now I just saw you have a new "common/ToastMessage.vue" as an component.
However..... does it really work? I don't seem to see message....or maybe it does?